src/components/NewMessage/NewMessage.vue

```diff
@@ -526,6 +536,15 @@ export default {
 			}
 		},
 
+		upload: {
+			immediate: true,
+			handler(value) {
+				if (value) {
+					this.silentChat = BrowserStorage.getItem('silentChat_' + this.token)
```

Comment:
This watcher is not needed anymore. But I would keep this line only and move to `currentUploadId` watcher:
- so if you set/reset silent toggle in UploadEditor, it got reflected in NewMessage

Reply:
Yes, that's why it was made in `upload` watcher in the first place. Isn't it better to keep in one watcher with `immediate` rather then handling the initial state in setup and the change in a watcher?

Reply:
`upload` is not a variable, and it's not changing with user actions. On the other hand, `currentUpload` is tracked in both inputs and actually represents opening/closing of modal